src/python/grpcio_channelz/grpc_channelz/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 The gRPC authors
```

Comment:
Please capitalize `Authors` here (and in other new files)

Reply:
Thanks!